src/client/languageParticipants.ts

```diff
@@ -63,8 +63,7 @@ export function getLanguageParticipants(): LanguageParticipants {
       return Array.from(languages) //
         .flatMap(language => {
           return [{
-            language,
-            scheme: 'file',
+            language
           }, {
             language,
             scheme: 'untitled',
```

Comment:
There are two entries in this array being returned. If we don't need to specify the scheme, then we can return just the one entry. In that case, we can replace the `flatMap` with a `map`.

Reply:
I did the change which is the same than vscode html.